### PR TITLE
examples: add LUIS.ai example

### PR DESCRIPTION
**Pull request by HcwXd**
This example shows how to integrate your bot with [LUIS.ai](https://luis.ai/).

### CODE REVIEW

**Comment by chentsulin** [comment]
Oops, lint failed. You can run `npm run lint:fix` to fix it.

**Comment by chentsulin** [comment]
/tmp/app/examples/with-luis.ai/index.js
  1:21  error    There should be at least one empty line between import groups                           import/order
  2:15  error    'axios' should be listed in the project's dependencies. Run 'npm i -S axios' to add it  import/no-extraneous-dependencies

**Comment by HcwXd** [comment]
I just figured out how to deal with that ESlint error, thanks!

File: examples/with-luis.ai/index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    const queryParams = {
+      verbose: true,
+      q: context.event.text,
+      'subscription-key': LUIS_APP_KEY,
+    };
+
+    const { data } = await axios.get(
+      `${LUIS_APP_ENDPOINT}luis/v2.0/apps/${LUIS_APP_ID}?${querystring.stringify(
+        queryParams
+      )}`
```
Comment (tw0517tw) [inline examples/with-luis.ai/index.js]: https://github.com/axios/axios/blob/master/README.md#L88-L93
`{ params: queryParams }` is better in readability

Comment (HcwXd) [reply examples/with-luis.ai/index.js]: Thanks for kindly pointing this out. It is definitely way more readable.

**Pull request merged**